### DISCUSSION TITLE
Add commands to create a new project

### DISCUSSION THREAD

**Discussion by Barbapapazes** [nuxtrdev/nuxtr-vscode#58] [Ideas]
Hello,

I was using the extension for a project without Nuxt. Then, I wanted to create a new docs inside my project. In order to do it, I would love to create a new Docus project inside a folder named `.docs`.

Being able to create a fresh project (starters and themes, https://nuxt.new/ or https://nuxt.new/themes) from any opened workspace could be a plus (could be done using the command palette or the side panel or both).

**Comment by adhamfarrag** [comment]
Hello @Barbapapazes, 

We'll make it happen soon :)

**Comment by adhamfarrag** [comment]
Hi @Barbapapazes, it took me couple months but finally it's here :) 
Eariler I rolled out a new version with a command `Nuxtr: Create new project` allowing you to clone starters prodvided on [Nuxt Starters](https://nuxt.new) or you can configure your own templates: 

```JSON
"nuxtr.projectTemplates": [
    {
        "name": "Vitesse",
        "description": " Vitesse for Nuxt 3 🏔💚⚡️ ",
        "repoURL": "https://github.com/antfu/vitesse-nuxt3",
        "branch": "main", // Optional
    },
],
```

Thanks for requesting this feature. 💚
